ocaml/parsing/jane_syntax.mli

```diff
@@ -23,6 +23,16 @@
 (*********************************************)
 (* Individual features *)
 
+(** An AST desc together with its attributes, including the attributes that
+    signal that the AST should be interpreted as a Jane Syntax AST.
+*)
+module With_attributes : sig
+  type 'desc t = 'desc Jane_syntax_parsing.With_attributes.t = private
```

Comment:
I know it seems gauche, but I'd be tempted to drop the reexport

Reply:
I don't think we can -- some call sites need to know that these types are equal. See anywhere that calls `wrap_desc`.